src/components/Filters/FilterRow/FilterRow.tsx

```diff
@@ -352,7 +351,6 @@ FilterRow.propTypes = {
     PropTypes.string,
     PropTypes.arrayOf(PropTypes.string),
     PropTypes.instanceOf(Date),
-    BaseDateRangePickerPropTypes,
```

Comment:
You shouldn't remove the prop type from here, because you will create a bunch of console errors. That's why I wrote that the custom proptypes should be removed only unless they are not used elsewhere.
In this case you have two options:
1. leave the `BaseDateRangePickerPropTypes` in place and remove it when you remove proptypes from FilterRow, FilterStatePropType and FieldPropTypes
2. remove proptypes where the `BaseDateRangePickerPropTypes` is used

Reply:
Rebased and fixed! 🙌